Lib/test/libregrtest/cmdline.py

```diff
@@ -333,6 +334,8 @@ def _create_parser():
                        help='enable Profile Guided Optimization (PGO) training')
     group.add_argument('--pgo-extended', action='store_true',
                        help='enable extended PGO training (slower training)')
+    group.add_argument('--tsan', dest='tsan', action='store_true',
+                       help='run a subset of test cases that are proper for the TSAN test')
```

Comment:
@swtaarrs It will be the more clear?

Reply:
Sorry, I completely missed this notification. Looks good to me :)